superset-frontend/src/datasource/DatasourceEditor.jsx

```diff
@@ -790,12 +790,12 @@ class DatasourceEditor extends React.PureComponent {
         </Fieldset>
         {this.allowEditSource && (
           <EditLockContainer>
-            <a href="#" onClick={this.onChangeEditMode}>
+            <span role="button" tabIndex={0} onClick={this.onChangeEditMode}>
```

Comment:
Does this still show the 👆 cursor on hover? If not, that would be nice :)

Reply:
Yes it does because of the button role.

Reply:
Yes it does because of `role="button"` and bootstrap. The file `scaffolding.less` contains the following:
```
[role="button"] {
  cursor: pointer;
}
```